src/codegen/compile.ml

```diff
@@ -8234,7 +8261,21 @@ and compile_exp (env : E.t) ae exp =
       compile_exp_vanilla env ae r ^^ set_r ^^
       FuncDec.ic_call env ts1 ts2 get_meth_pair get_arg get_k get_r add_cycles
       end
-
+    | ICCallRawPrim, [p;m;a;k;r] ->
+      SR.unit, begin
+      let (set_meth_pair, get_meth_pair) = new_local env "meth_pair" in
+      let (set_arg, get_arg) = new_local env "arg" in
+      let (set_k, get_k) = new_local env "k" in
+      let (set_r, get_r) = new_local env "r" in
+      let add_cycles = Internals.add_cycles env ae in
+      compile_exp_vanilla env ae p ^^
+      compile_exp_vanilla env ae m ^^
+      Tuple.from_stack env 2 ^^ set_meth_pair ^^
```

Comment:
Here you need to turn the text into a blob, because above we have
```
(* The method name *)
      get_meth_pair ^^ Arr.load_field 1l ^^ Blob.as_ptr_len env ^^
```

Reply:
Excellent, thanks. That's precisely what I was worried about.

Reply:
Fixed!